src/util/postgres_client.js

```diff
@@ -610,7 +610,7 @@ class PostgresTable {
         if (!process.env.CORETEST && !process.env.NC_NSFS_NO_DB_ENV) {
             // Run once a day
             // TODO: Configure from PostgreSQL
-            setInterval(this.vacuumAndAnalyze, 86400000, this).unref();
+            setInterval(this.vacuumAndAnalyze, config.VACCUM_ANALYZER_INTERVAL, this).unref();
```

Comment:
I think you should also remove the throw err that is in the catch of vacuumAndAnalyze, to avoid crashing the process in any case.